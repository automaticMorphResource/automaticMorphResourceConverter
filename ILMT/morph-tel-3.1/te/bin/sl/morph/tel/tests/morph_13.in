1	vAwAvaraNa	unk	
1	kAluRyaM	unk	
1	,	unk	
1	saraxA	unk	
1	kosaM	unk	
1	cepalu	unk	
1	pattataM	unk	
1	,	unk	
1	plAstik	unk	
1	vyarWAla	unk	
1	valla	unk	
1	I	unk	
1	rIP	unk	
1	loni	unk	
1	prANi	unk	
1	prapaMcaM	unk	
1	koVMwa	unk	
1	meraku	unk	
1	naSiMci	unk	
1	powoMxi	unk	
1	.	unk	
