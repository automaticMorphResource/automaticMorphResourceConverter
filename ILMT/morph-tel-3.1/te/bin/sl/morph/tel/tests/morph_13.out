1	vAwAvaraNa	unk	<fs af='vAwAvaraNaM,n,,sg,,o,ti,ti'>
2	kAluRyaM	unk	<fs af='kAluRyaM,n,,sg,,d,0,0'>
3	,	unk	<fs af='&comma;,punc,,,,,,'>
4	saraxA	unk	<fs af='saraxA,n,,sg,,d,0,0'>|<fs af='saraxA,n,,sg,,o,ti,ti'>|<fs af='saraxu,n,,sg,,,0,0_A'>
5	kosaM	unk	<fs af='kosaM,avy,,,,,0,0_avy'>
6	cepalu	unk	<fs af='cepa,n,,pl,,d,0,0'>|<fs af='cepa,n,,pl,,,vu,vu'>
7	pattataM	unk	<fs af='pattu,v,any,any,any,,adaM,adaM'>
8	,	unk	<fs af='&comma;,punc,,,,,,'>
9	plAstik	unk	<fs af='plAstik,n,,sg,,d,0,0'>|<fs af='plAstik,n,,sg,,d,0,0'>|<fs af='plAstik,n,,sg,,o,ti,ti'>|<fs af='plAstik,n,,sg,,o,ti,ti'>
10	vyarWAla	unk	<fs af='vyarWaM,n,,pl,,o,ti,ti'>
11	valla	unk	<fs af='valla,avy,,,,,0,0_avy'>
12	I	unk	<fs af='I,avy,,,,,,'>|<fs af='I,avy,,,,,,'>
13	rIP	unk	<fs af='rIP,n,,sg,,d,0,0'>|<fs af='rIP,n,,sg,,o,ti,ti'>
14	loni	unk	<fs af='loni,avy,,,,,,'>|<fs af='lona,nst,,,,,yoVkka,yoVkka_adv'>
15	prANi	unk	<fs af='prANi,n,,sg,,d,0,0'>|<fs af='prANi,n,,sg,,o,ti,ti'>
16	prapaMcaM	unk	<fs af='prapaMcaM,n,,sg,,d,0,0'>
17	koVMwa	unk	<fs af='koVMwa,pn,,sg,,d,0,0'>|<fs af='koVMwa,n,,sg,,d,0,0'>|<fs af='koVMwa,n,,sg,,o,ti,ti'>
18	meraku	unk	<fs af='meraku,avy,,,,,0,0_avy'>|<fs af='mera,n,,sg,,,ki,ki'>
19	naSiMci	unk	<fs af='naSiMcu,v,any,any,any,,i,i'>
20	powoMxi	unk	<fs af='po,v,fn,sg,3,,wunn,wunn'>
21	.	unk	<fs af='&dot;,punc,,,,,,'>
